website/docs/migrations/migrate-form-ckb-sdk-utils.md

```diff
@@ -0,0 +1,1483 @@
+# Migrate From ckb-sdk-utils to Lumos
+
+## Summary
+In order to merge ckb-sdk-js and lumos ecology, the funtion of ckb-sdk-utils will be migrated to lumos, most of the function can be replaced by the corresponding functions directly, while the normolize.ts need to be replaced by lumos/codec
```

Comment:
In order to unify CKB development ecology, the functions of `ckb-sdk-utils` will be migrated to `@ckb-lumos/lumos`, most of the functions can be replaced by the corresponding functions directly, but the `normalize.ts` need to be rewriten with `@ckb-lumos/codec`.